django/cantusdb_project/main_app/views/views.py

```diff
@@ -236,14 +236,15 @@ def csv_export(request, source_id):
         ]
     )
     for entry in entries:
```

Comment:
Do you need to include this in the loop? Because we are using the source siglum, it will always be the same for each entry? 

I am just wondering if this is a case that will introduce a lot of duplicate queries?

Reply:
Good call

Reply:
@lucasmarchd01 - I see we're setting `diff_db = entry.diff_db.id if entry.diff_db else ""`, but then we're writing `entry.diff_db` rather than the value of the `diff_db` variable. I can't recall the specifics, but weren't you trying to fix something lately related to differentiae and the CSV export view? Might this be relevant?

Reply:
Hmm...interesting. Approving cause the bit about the siglum looks good to me, but this certainly seems like something to investigate.